### PR TITLE
Updating the info for private dns zone being customer managable.

### PR DESCRIPTION
**Pull request by jmighion**
This PR adds information about the AKS Private DNS zone, the customers soon having the ability to partially manage the Private DNS zones in the managed resource group, and cleans up the capitalization to match Azure docs.

Draft for now until we're live with these changes.

### CODE REVIEW

**Comment by ariordan-redhat** [comment]
@jmighion Some rephrasing suggestions; ping me if you have questions.

**Comment by jmighion** [comment]
Thanks for the fixes! This is ready, just waiting on production release before marking it as ready to merge. Should be sometime this week or early next week.

File: stories/assembly-azure-support.adoc [hunk 1]
```diff
@@ -13,26 +13,37 @@ Due to the architecture of the application and the deployment strategy in Azure,
 As an {AAPonAzureNameShort} user, you can configure the {PlatformNameShort} network to peer your own network.
 By doing so, you can grant access from the {PlatformNameShort} instance to all the assets associated with your own network that you want to manage.
 Also, you can route all the {PlatformNameShort} traffic to your own Virtual Network Appliances to control, audit, or block traffic from the {PlatformNameShort} instance to the internet.
-To do this, you must consider the URLs that need to be allowlisted for {PlatformNameShort} to work properly. 
+To do this, you must consider the URLs that need to be allowlisted for {PlatformNameShort} to work properly.
```
Comment (ianf77) [inline stories/assembly-azure-support.adoc:16]: URLs that must be

File: stories/assembly-azure-support.adoc [hunk 2]
```diff
@@ -13,26 +13,37 @@ Due to the architecture of the application and the deployment strategy in Azure,
...
 
 {AAPonAzureNameShort} uses Azure's managed DNS services when deployed.
 
-To use private DNS records that cannot be resolved publicly, you can either use Azure Private DNS Zones that are peered to the managed application VNET, or you can make a submit request to Red Hat to submit DNS zones that should be forwarded to a customer-managed private DNS server.
+To use private DNS records that cannot be resolved publicly, you can either use Azure Private DNS zones that are peered to the managed application VNET, or you can make a submit request to Red Hat to submit DNS zones that should be forwarded to a customer-managed private DNS server.
 
-A limitation of Private DNS Zones is that only one instance of a given zone may be linked to a Virtual Network.
-Attempting to link zones that match the names of Private DNS Zones in the managed resource group will cause conflicts.
+A limitation of Private DNS zones is that only one instance of a given zone may be linked to a Virtual Network.
+Attempting to link zones that match the names of Private DNS zones in the managed resource group will cause conflicts.
```
Comment (ianf77) [inline stories/assembly-azure-support.adoc:27]: resource group causes conflicts.

File: stories/assembly-azure-support.adoc [hunk 3]
```diff
@@ -13,26 +13,37 @@ Due to the architecture of the application and the deployment strategy in Azure,
...
 
-Failure to properly maintain the records in the Private DNS Zone can prevent the managed application from operating.
+Failure to properly maintain the records in the Private DNS zone can prevent the managed application from operating.
 
-For more information about working with Private DNS Zones, see the link:https://access.redhat.com/articles/6983525[Private DNS with Red Hat Ansible Automation Platform on Microsoft Azure] article on the Red Hat customer portal.
+The AKS Private DNS zone cannot be customer managed and still allow Red Hat to update or upgrade the managed AKS that is a part of this offering.
+To allow Red Hat to upgrade the customer AKS to the latest version during the maintenance windows, do not unlink the `<GUID>.privatelink.<region>.azmk8s.io` Private DNS zone.
+For more information on this limitation, see https://learn.microsoft.com/en-us/troubleshoot/azure/azure-kubernetes/create-upgrade-delete/createorupdatevirtualnetworklinkfailed-error["CreateOrUpdateVirtualNetworkLinkFailed" error when updating or upgrading an AKS cluster] in the Microsoft Azure documentation.
```
Comment (ianf77) [inline stories/assembly-azure-support.adoc:37]: see link:https.....

**Review by ianf77 — COMMENTED** [review]
A few comments.

**Comment by jmighion** [comment]
@ianf77 Thanks for the review. Made the changes. Trying to get this merged today if possible. Anything else need updating?

**Comment by ianf77** [comment]
I'll run through it again, but if it's clear do you want it merging?

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**